misc/test_site/mac_addrs/3c5ab41e8f0b/module_config.json

```diff
@@ -5,7 +5,7 @@
       "enabled": true
     },
     "macoui": {
-      "enabled": true,
```

Comment:
why disable this?

Reply:
I'm disabling the tests that need work because either they're flaky or are missing something important.